tests/PHPStan/Rules/Functions/CallToFunctionParametersRuleTest.php

```diff
@@ -635,10 +635,6 @@ public function testArrayUdiffCallback(): void
 				'Parameter #3 $data_comp_func of function array_udiff expects callable(1|2|3|4|5|6, 1|2|3|4|5|6): int, Closure(string, string): string given.',
 				6,
 			],
-			[
-				'Parameter #3 $data_comp_func of function array_udiff expects callable(1|2|3|4|5|6, 1|2|3|4|5|6): int, Closure(int, int): (literal-string&lowercase-string&non-falsy-string&numeric-string) given.',
```

Comment:
Why did this error disappear?

Reply:
Looks like it was because I did not collect the accessory string types correctly; corrected by f319d828c but the error message no longer has `lowercase-string` in it, which I *think* is right but please correct me as needed. @VincentLanglet your thoughts?

Reply:
To me, you're loosing the `lowercase-string` because of
```
if ($lower && $upper) {
			return;
		}
```

Which means that "25" is lowercase-string&uppercase-string but `["25", "26"]` is not considered as `array<lowercase-string&uppercase-string>` anymore which seems wrong and will lead to false positive I think. (for same reason than https://github.com/phpstan/phpstan-src/pull/3613#discussion_r1835754973) 

I would say accessory need to be kept. 
Maybe we will want to simplify the `description` for errors later ; but with the changes you made, it's not the description you changed, you just lost the accessory type. So to me the patch is not correct.

Maybe the accessory were lost because of the error I already mentionned, like the fact
- AccessoryLowercaseStringType::isUppercaseString is `TrinaryLogic::createMaybe` (and not No)
- AccessoryUppercaseStringType::isLowercaseString is `TrinaryLogic::createMaybe` (and not No)

Reply:
@VincentLanglet Very good, I will address the createMaybe() comment next. Thanks for your patience.

Reply:
@VincentLanglet I have addressed createMaybe() in d472b913b and updated the relevant uppercase-string tests; however, the corollary lowercase-string tests did *not* need updating, which makes me think something else might be missing. Your thoughts?